modules/nested-accordion/assets/js/frontend/handlers/nested-accordion.js

```diff
@@ -62,7 +62,14 @@ export default class NestedAccordion extends Base {
 		event.preventDefault();
 
 		const accordionItem = event.currentTarget.parentElement,
-			accordionContent = accordionItem.querySelector( '.e-n-accordion-item > .e-con' );
```

Comment:
This change is not related to this PR,
Jus replacing a hard-coded selector with a predefined var we have.